src/index.js

```diff
@@ -33,9 +33,13 @@ const path = require("path")
 const { setAliases } = require("require-control")
 
 // Force resolution of potentially `yarn link`'d modules to the local node_modules
-// folder. This gets around SSR issues involving single react context requirements.
-// This is server-side only. Client-side must be resolved via webpack.
+// folder. This gets around SSR issues involving single react context requirements,
+// amongst other things. This is server-side only. Client-side must be resolved
+// via webpack.
 setAliases({
+  "express-http-context": path.resolve(
```

Comment:
Since we're sharing context values across application boundaries, we need to be sure it still works while `yarn linking`. `require-control` _forces_ the node process to look in a specific location for packages. Very useful lib that solved a lot of earlier `yarn link` problems.